### PR TITLE
refactor: replace @hapi/shot with light-my-request

### PR DESCRIPTION
**Pull request by shubham201**
Addressing #6391
## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by dhmlau — APPROVED** [review]
@shubham201, thanks for your PR. LGTM. Would like to get another review/approval before merging. Thanks.

**Review by samarpanB — APPROVED** [review]

**Comment by samarpanB** [comment]
LGTM. However, you may need to rebase this PR as there are some conflicts showing up.

**Comment by shubham201** [comment]
> LGTM. However, you may need to rebase this PR as there are some conflicts showing up.
@samarpanB 
Deleted my changes by mistake while syncing light-my-request branch to the current master branch. 🤦🏻‍♂️

**Pull request closed without merging**
